src/client.cpp

```diff
@@ -185,7 +185,7 @@ void ADDON_ReadSettings(void)
   else
     g_strTimeshiftBufferPath = DEFAULT_TSBUFFERPATH;
 
-  // read setting "enabletimeshift" from settings.xml 
+  // read setting "readtimeout" from settings.xml 
```

Comment:
I'm not a big fan of comments just stating what is obvious.

Reply:
Done